### PR TITLE
 OSPK8-352 Switch to OSP16.2 as default

### PR DESCRIPTION
**Pull request by stuggi**
Also removes low-memory-usage.yaml to speed up tempest test runs.

Depends-On: https://github.com/openstack-k8s-operators/osp-director-operator/pull/393

### CODE REVIEW

**Review by abays — APPROVED** [review]
/lgtm

File: ansible/vars/default.yaml [hunk 1]
```diff
@@ -196,7 +196,7 @@ osp:
     external: enp6s0
     ospnetwork: enp7s0
   network_type: vlan
-  container_tag: 17.0_20210907.1
+  container_tag: 16.2_20210901.2
```
Comment (dprince) [inline ansible/vars/default.yaml:199]: perhaps in another commit we can support the publicly available OSP images (not our operator... but the OSP ones). As is this is going to still pull from registry-proxy.engineering.redhat.com/rh-osbs so internal only... but its okay for now.

Comment (stuggi) [reply ansible/vars/default.yaml:199]: yes, in addition to the 16.2/17.0 we can also provide var files for upstream versions, and then default to one of them. I'll create a jira task for that

**Comment by stuggi** [comment]
CI fails because of missing fencing patch for:

~~~
2021-09-24T08:06:18.753Z	INFO	controllers.OpenStackControlPlane	Underlying OSVMSet controller hit an error: secrets "osp-director-operator-kubevirtfencing" not found
~~~

**Pull request merged**